src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -639,7 +639,7 @@ static int flush_send(int dst, int vni, MPIDI_OFI_dynamic_process_request_t * re
 
     MPIR_Comm *comm = MPIR_Process.comm_world;
     fi_addr_t addr = MPIDI_OFI_av_to_phys(MPIDIU_comm_rank_to_av(comm, dst), vni, vni);
-    int data = 0;
+    static int data = 0;
```

Comment:
Using `static` everywhere is not very nice. It is scoped, but essentially it's a global variable.
If we can send a random data here (e.g., `req->tag` or any global value), that'd be great.

Reply:
After the discussion, I believe using `static` here is valid. Sorry. I will approve this PR.

Reply:
I think using static is good. It is the other branch (the recv) that uses an arbitrary field from a different purpose raises a bad smell, but I don't have a better solution for it.